### PR TITLE
Messaging

### PR DESCRIPTION
**Pull request by Dancodes92**
still needs styling

### CODE REVIEW

File: script/seed.js [hunk 1]
```diff
@@ -24,8 +24,6 @@ async function seed() {
     email: "test@test.com",
     password: "123",
     gender: "Cat",
-    genderCategory: "Woman",
```
Comment (VioletLeon) [inline script/seed.js]: Will need to be added back

File: script/seed.js [hunk 2]
```diff
@@ -37,20 +35,12 @@ async function seed() {
   for (let i = 0; i < 10; i++) {
     let randomAge = Math.floor(Math.random() * 20 + 18)
 
-    const genderCategory = ["Man", "Woman"]
```
Comment (VioletLeon) [inline script/seed.js]: Will need to be added back

**Review by VioletLeon — APPROVED** [review]
Looks good to me

**Comment by VioletLeon** [comment]
But I'm too afraid to merge, so someone else please take a look :D

**Review by somecallmehans — APPROVED** [review]
Approved!

**Pull request merged**